gef-ui/templates/partials/unissued-facilities.njk

```diff
@@ -34,6 +34,10 @@
     </div>
   </div>
 
+{% if successMessage %}
```

Comment:
Success message is rendered by `mojBanner` component bellow.

These 3 new lines don't do anything, because we don's pass parameter `successMessage`. Please remove 3 new lines.

Reply:
Removed 3 lines